.github/workflows/amplify_db_common.android.yaml

```diff
@@ -0,0 +1,28 @@
+# Generated with aft. To update, run: `aft generate workflows`
+name: amplify_db_common Android
```

Comment:
Can we remove the /android folder from amplify_db_common?

Reply:
Oh I see there is a native android dep for eu.simonbinder:sqlite3-native-library:3.40.0. nvm.